ensure-is-media.js

```diff
@@ -9,16 +9,57 @@ let makeUseIsMedia = (
 // improving the algorithms inside, etc, see this:
 // https://github.com/viewstools/morph/blob/master/ensure-is-media.js
 
-import { useMedia } from 'use-media';
+import { useState, useEffect } from 'react'
+
+const MOCK_MEDIA_QUERY_LIST = {
+  media: '',
+  matches: false,
+  onchange: () => {},
+  addListener: () => {},
+  removeListener: () => {},
+  addEventListener: () => {},
+  removeEventListener: () => {},
+  dispatchEvent: _ => true, // dispatchEvent: (_: Event) => true
+}
+
+let useMedia = (
+  query,
+  defaultState = window.matchMedia(query).matches
```

Comment:
This will fail if `window.matchMedia` doesn't exist.
What about refactoring the mock like this:
```
function matchMedia(query) {
  if (typeof window === 'undefined') return MOCK_MEDIA_QUERY_LIST

  return window.matchMedia(query)
}
```

That way the rest of the uses can be simplified to:
```
function useMedia(query, defaultState = matchMedia(query).matches))
```

and
```
useEffect(() => {
let mediaQueryList = matchMedia(query)
```

Reply:
Does this need to be `Boolean(matchMedia(query)).matches` from what I'm seeing below?